README.md

```
@@ -118,24 +116,15 @@ src/
-> *(Agrega aquí imágenes del sitio usando markdown)*  
-> `![Login](./screenshots/login.png)`  
-> `![Feed](./screenshots/feed.png)`  
-> `![Registro](./screenshots/register.png)`
-
+## OUTDATED SECTION ------------ END -------------
 ---
 
-## 👨‍💻 Autores
+## 🚀 TOP CONTRIBUIDORES
+<a href="https://github.com/eestn2/UNiTEC/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=eestn2/UNiTEC" />
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)